applications/basic_networking_ping/cpp/CMakeLists.txt

```diff
@@ -31,11 +31,11 @@ target_link_libraries(basic_networking_ping
 
 # Copy config file
 add_custom_target(basic_networking_ping_rx_yaml
-  COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_CURRENT_SOURCE_DIR}/../basic_networking_ping_rx.yaml" ${CMAKE_CURRENT_BINARY_DIR}
+  COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_CURRENT_SOURCE_DIR}/../basic_networking_ping_rx.yaml" ${CMAKE_BINARY_DIR}
   DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/../basic_networking_ping_rx.yaml"
 )
 add_custom_target(basic_networking_ping_tx_yaml
-  COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_CURRENT_SOURCE_DIR}/../basic_networking_ping_tx.yaml" ${CMAKE_CURRENT_BINARY_DIR}
+  COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_CURRENT_SOURCE_DIR}/../basic_networking_ping_tx.yaml" ${CMAKE_BINARY_DIR}
```

Comment:
Same for this.

Reply:
This change allows the user to launch the application using `./devcontainer build_and_run` and specify the config file as put in the README.

Reply:
Fixed as per the recommendation and updated `metadata.json` to use `holohub_app_bin` so it can pick up the config file correctly.